core-web/apps/dotcms-ui/src/app/api/services/guards/ema-app/edit-page.guard.ts

```diff
@@ -29,7 +29,7 @@ function getUpdatedQueryParams(route: ActivatedRouteSnapshot) {
     const newQueryParams = {
         ...route.queryParams,
         language_id: route.queryParams.language_id || 1,
-        url: route.queryParams.url.substring(1) || 'index',
+        url: route.queryParams.url || 'index',
```

Comment:
I kinda did this because we don't know if this can come with our without the `/` we should try to support both.

Reply:
This `.substring(1)` method was removing the first character without checking if it was a `/`.

Note: We remove the `/` if any in the [DotPageApiService](https://github.com/dotCMS/core/blob/e7d631593fe9059b64c1ca38fad156bfde9d7f71/core-web/libs/portlets/edit-ema/portlet/src/lib/services/dot-page-api.service.ts#L72-L77)

Reply:
What if we simply remove the first `/` from the two URLs we are comparing in `getSecretByUrlMatch` so we don't have to modify any parameters?

https://github.com/dotCMS/core/blob/e7d631593fe9059b64c1ca38fad156bfde9d7f71/core-web/libs/data-access/src/lib/ema-app-configuration/ema-app-configuration.service.ts#L90-L112